rocAL/rocAL/include/augmentations/geometry_augmentations/node_crop.h

```diff
@@ -29,13 +29,14 @@ THE SOFTWARE.
 class CropNode : public Node
 {
 public:
-    CropNode(const std::vector<Image *> &inputs, const std::vector<Image *> &outputs);
+    CropNode(const std::vector<rocalTensor *> &inputs, const std::vector<rocalTensor *> &outputs);
     CropNode() = delete;
+    ~CropNode();
     void init(unsigned int crop_h, unsigned int crop_w, float x_drift, float y_drift);
     void init(unsigned int crop_h, unsigned int crop_w);
     void init( FloatParam *crop_h_factor, FloatParam *crop_w_factor, FloatParam * x_drift, FloatParam * y_drift);
-    unsigned int get_dst_width() { return _outputs[0]->info().width(); }
-    unsigned int get_dst_height() { return _outputs[0]->info().height_single(); }
+    unsigned int get_dst_width() { return _dest_width; }
+    unsigned int get_dst_height() { return _dest_height; }
```

Comment:
We are supposed to send height single ryt? _outputs[0]->info().max_shape()[1] 
I think this will not give height single.

Reply:
Changed this
Removed the _dest_width and _dest_height as it was not used